### ISSUE TITLE
Energy Transmitter issues

### ISSUE THREAD

**Issue by notnate75** [RuochenFu21/CreateEnderTransmission#12]
so i have been running the energy transmitter for 2 weeks on a 1.20.1 server and it has worked just fine.  i am the only one on my server so far running this, and the only other one who might have the ability, is not (i have confirmed); so there cannot be a "channel" cross talk issue.  however, the other day i made a chunk loader and since connecting it to the system it started breaking my steam engine during periods without anyone logged into the server, forcing me to break down the steam engines on the overall contraption and replace them to start it back up.  one thing i noticed right away, that did not seem to be a problem until i tried to use the chunk loader, was that i had a rotational speed controller coming off of my steam engine contraption and feeding the energy transmitter which was right next to one of the steam engines then causing the overall contraption to run at max speed (this is due to the transmitter giving off rotation from ANY face i realized eventually).  the problem now is that it is maintaining its previous connections and/or speeds even when broken and replaced (i was actually running everything last night without the main input node connected because of this, and the steam engine was still showing the same SU used as if it was).  I've tried changing the channel, but it does not retain the channel data when broken and returns to channel 1 when placed; so they are always in an overstressed state now as soon as placed and just breaks anything i try to connect them to since their direction/speed or other variables do not match.

**Comment by furuochen-dev** [comment]
TL;DR Duplicate #8 I think
